src/main/java/com/manardenza/console/ConsoleMain.java

```diff
@@ -2,15 +2,17 @@
 
 public class ConsoleMain {
 
-    private static ConsoleMain instance;
+    private ContentsUserMenu contentsUserMenu;
 
-    private ConsoleMain() {
+    public ConsoleMain(ContentsUserMenu contentsUserMenu) {
+        this.contentsUserMenu = contentsUserMenu;
     }
 
-    public static ConsoleMain getInstance() {
-        if (instance == null) {
-            instance = new ConsoleMain();
+    public void consoleMain() {
+        ListMenu.makeMenus();
+        while (true) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Add an end condition to this loop. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2189)